src/core/client/admin/components/UserRole/UserRoleChange.tsx

```diff
@@ -6,6 +6,8 @@ import React, {
   useState,
 } from "react";
 
+import { validateRoleChange } from "coral-common/permissions";
```

Comment:
Can you please remove the empty line below this to organize with the other coral imports?

Reply:
this doesnt say outdated, but I'm not seeing an empty line below this import.